openpype/settings/defaults/project_settings/ftrack.json

```diff
@@ -493,7 +493,29 @@
             "upload_reviewable_with_origin_name": false
         },
         "IntegrateFtrackFarmStatus": {
-            "farm_status_profiles": []
+            "farm_status_profiles": [
+                {
+                    "hosts": [
+                        "celaction"
```

Comment:
Is this a typo or what is `celaction` ?

Reply:
It's one of the supported hosts of openpype, see [celaction](https://github.com/ynput/OpenPype/tree/develop/openpype/hosts/celaction) hosts folder.

I think it's this [celaction](https://www.celaction.com/).